### PR TITLE
feat: explain how to push to prod

### PR DESCRIPTION
**Pull request by nchabanoles**
Add a How To explaining the configuration to make to deploy properly according to each situation.

### CODE REVIEW

File: md/how_to_push_LA_to_prod.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+Wether the deployment is managed by your CIs jobs or your administrator, is up to you.
+
+
+## Pre-requisites
+
+Deploying to a production environment (or any sensible environment) requires extra care. Before goind through this tutorial, please make sure you have read and understood the concepts described in these pages:
+* [Deploy Living App artifacts](livingapp_deploy.md)
+* [Build and deploy (Best Practices)](livingapp_build_and_deploy.md)
+
+## Notion of projects
```
Comment (nchabanoles) [inline md/how_to_push_LA_to_prod.md]: Maybe this whole section dealing with projects could be deleted. I am not sure if it helps...
Just the examples below might be enough for a user to understand what he / she has to do to deal with deployment when it comes to production.

**Comment by tnguyen1** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

**Comment by uguy** [comment]
BCD refactoring in progress, no sure yet if the user will use it this way.

**Comment by uguy** [comment]
won't merge, stack is removed.

**Pull request closed without merging**